### DISCUSSION TITLE
第2节课笔记(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#109] [General]
请将第二节课的笔记链接上传到这里，并且记得在17班的表格中也填写！

**Comment by Alisa742** [comment]


**Reply by Jianfeng777** [reply]
作业请传到另外一个discussion中！

**Comment by fine-lab** [comment]
第二节课笔记：https://blog.csdn.net/SmallZCan_t/article/details/135443841?spm=1001.2014.3001.5502

**Comment by Alisa742** [comment]
笔记连接：https://blog.csdn.net/weixin_53222965/article/details/135438867?spm=1001.2014.3001.5501

**Comment by l241025097** [comment]
![02《轻松玩转书生浦语大模型趣味Demo》读书笔记](https://github.com/InternLM/tutorial/assets/31737569/4650b1e4-7f26-4b65-a183-5edb5caecd4b)

**Comment by zcg-joker** [comment]
[第二节课笔记](https://blog.csdn.net/m0_55764641/article/details/135452507)

**Comment by AxrLH** [comment]
笔记+作业：https://blog.csdn.net/Axr_Samuel/article/details/135456433?spm=1001.2014.3001.5501

**Comment by langlibai66** [comment]
笔记： http://t.csdnimg.cn/uze5H

**Comment by xaiojianjianbujiandan** [comment]
第二节课笔记：http://t.csdnimg.cn/yDAEp

**Comment by EthanHan1999** [comment]
第二节课笔记：https://blog.csdn.net/weixin_44015617/article/details/135483859?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135483859%22%2C%22source%22%3A%22weixin_44015617%22%7D

**Comment by popcornGit** [comment]
第二节课笔记
https://blog.csdn.net/weixin_38771145/article/details/135489747

**Comment by JACK626** [comment]
第2节课笔记 https://www.yuque.com/xiaotuanyue/mdr8d5/hwvcg4fq41elo7ec?singleDoc# 《第二节课笔记+作业》

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/676932341

**Comment by StarDuster** [comment]
第二次课笔记 https://gist.github.com/StarDuster/0b76bb0c821fc0f08e445dbf9489d233
